package.json

```diff
@@ -21,7 +21,7 @@
     "bootstrap-switch": "3.3.2",
     "colors": "1.1.2",
     "css-loader": "^0.26.1",
-    "eonasdan-bootstrap-datetimepicker": "~4.15",
+    "eonasdan-bootstrap-datetimepicker": "~4.17",
```

Comment:
Is there a reason to upgrade this? It broke one of the tests due to a date formatting change.